docs/sources/datasources/aws-cloudwatch/query-editor/index.md

```diff
@@ -45,10 +45,10 @@ You can build two types of queries with the CloudWatch query editor:
 
 ### Create a Metric Search query
 
-To create a valid Metric Search query, specify the namespace, metric name, and at least one statistic.
+To create a valid Metric Search query, specify the namespace, metric name, and at least one statistic. Dimensions are optional, but for a dimension to be considered part of the query it must have both a key and a value.
```

Comment:
Rather than adding a table I added to the explanation, since the * vs empty bit is just that dimensions with no value are not considered part of the query, and one row isn't really a table.